astropy_helpers/openmp_helpers.py

```diff
@@ -13,15 +13,17 @@
 
 from __future__ import absolute_import, print_function
 
+import datetime
+import glob
 import os
+import subprocess
 import sys
-import glob
 import tempfile
-import subprocess
```

Comment:
Please don't shuffle the imports around but just add the new ones, they were sorted by length on purpose.

Reply:
Woops, sorry, probably an auto-format.

Reply:
Do you mean import name length? If so, I should insert the two additions accordingly, right, rather than just append them to the list?